torchaudio/functional.py

```diff
@@ -430,6 +431,8 @@ def create_fb_matrix(
         f_max (float): Maximum frequency (Hz)
         n_mels (int): Number of mel filterbanks
         sample_rate (int): Sample rate of the audio waveform
+        norm (Optional[str]): If 'slaney', divide the triangular mel weights by the width of the mel band
```

Comment:
Do we expected further normalization schemes? Does it make sense to split these normalizations out into its own layer? Are they useful in other contexts (maybe for volume normalization and such)?

Reply:
> Do we expected further normalization schemes? 

Yes, we could, see librosa/librosa#1050. Librosa itself is in process of adding other normalization as mentioned in this pull request.

> Does it make sense to split these normalizations out into its own layer? 

Not according to [this comment](https://github.com/pytorch/audio/pull/426#discussion_r375998053).

> Are they useful in other contexts (maybe for volume normalization and such)?

The normalization is done against `f_pts` which is computed within `create_fb_matrix`. I'm not aware of other use case.